src/components/ExpiryNotification/ExpiryNotificationModal.js

```diff
@@ -0,0 +1,138 @@
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import { EmailComponent } from '@buidlhub/buidlhub-ens-notifications'
+
+import {
+  default as Button,
+  getButtonDefaultStyles,
+  getButtonStyles
+} from '../Forms/Button'
+import GlobalState from '../../globalState'
+import Loader from '../Loader'
+import Modal from '../Modal/Modal'
+import mq from 'mediaQuery'
+import { ReactComponent as Email } from '../Icons/Email.svg'
+import TruncatedAddress from './TruncatedAddress'
+
+const BUILDHUB_LINK =
+  'https://buidlhub.com/?utm_source=ens&utm_medium=link&utm_campaign=reminder'
+const EXPIRY_NOTIFICATION_MODAL_NAME = 'buidlhub-expiry-notification'
+const INPUT_NAME = 'buidlhub-email-input'
+
+// Custom components to match styling & use media queries
+const LoadingComponent = styled(Loader)`
+  display: inline-block;
+  margin: 0 10px;
+`
+
+const ModalIconContainer = styled('div')`
+  display: none;
+  justify-self: end;
+  ${mq.xLarge`
+    display: block;
+  `}
+`
+
+const FormComponent = styled('form')`
+  display: grid;
+  grid-gap: 10px;
+  grid-template-columns: 1fr;
+  ${mq.xLarge`
+    grid-template-columns: 2fr 1fr;
+  `}
+`
+
+const LabelComponent = ({ address, name }) => {
+  return (
+    <>
+      <div
+        className={css`
+          font-weight: 200;
+        `}
+      >
+        <a href={BUILDHUB_LINK} target="_blank" rel="noopener noreferrer">
+          BUIDLHub
+        </a>{' '}
+        checks if address <TruncatedAddress address={address} /> has expiring
+        names within 30 days and sends you a reminder email every week.
+        <label
+          htmlFor={name}
+          className={css`
+            display: block;
+            margin-top: 20px;
+          `}
+        >
+          Enter email to receive email notifications.
+        </label>
+      </div>
+      <ModalIconContainer>
+        <Email />
+      </ModalIconContainer>
+    </>
+  )
+}
+
+const buttonStyles = `
+  margin: 5px;
+  position: relative;
+  float: right;
+`
+
+const CancelComponent = styled(Button)`
+  ${getButtonDefaultStyles()}
+  ${getButtonStyles({ type: 'hollow' })}
+  ${buttonStyles}
+ `
+
+const SubmitComponent = styled(Button)`
+    ${getButtonDefaultStyles()}
+    ${getButtonStyles({ type: 'primary' })}
+    ${buttonStyles}
+`
+
+const ExpiryNotificationModal = () => {
+  const { toggleModal, currentModal } = useContext(GlobalState)
+  const { t } = useTranslation()
+
+  const address = currentModal ? currentModal.address : null
+
+  // Passing a references domain name may be useful for future capabilities, if a
+  // user wants to monitor ENS names owned by others for upcoming expiration. This
+  // feature would require additional planning / coordination between ENS <> BUIDLHub.
+
+  // const domainName = currentModal ?
+  //   currentModal.domainName :
+  //   null;
+
+  const translation = {
+    cancel: t(`c.cancel`),
+    submit: t(`c.save`),
+    close: 'Close',
```

Comment:
Please add translation strings for all of these `en.json`

Reply:
And after please run `yarn i18n` to duplicate the keys in non English languages

Reply:
Looks like "Close" would be good in `c.close` and the rest would need a namespace such as `emailReminder.placeholder`

Reply:
* close - Removed this completely after prior discussion w. @makoto 
* placeholder - moved  to `expiryNotification.placeholder`- https://github.com/ensdomains/ens-app/pull/776/commits/481c5e84c885cccccad7408b4893a25dc48444c8